src/app/+item-page/full/full-item-page.component.html

```diff
@@ -29,6 +29,11 @@
       <ds-item-page-full-file-section [item]="item"></ds-item-page-full-file-section>
       <ds-item-page-collections [item]="item"></ds-item-page-collections>
       <ds-item-versions class="mt-2" [item]="item"></ds-item-versions>
+      <div class="button-row bottom" *ngIf="fromWfi">
+        <div class="text-right">
+          <a class="btn btn-outline-secondary mr-1" (click)="back()"><i class="fas fa-arrow-left"></i> {{'item.page.return' | translate}}</a>
```

Comment:
```suggestion
          <a class="btn btn-outline-secondary mr-1" href="javascript:void(0)" (click)="back()"><i class="fas fa-arrow-left"></i> {{'item.page.return' | translate}}</a>
```

An `<a>` tag needs an `href`, otherwise it isn't accessible by a screenreader. You can also tell because the hover color doesn't work if you omit the href.